File: problem_sets/mario.c
#include <cs50.h>
#include <stdio.h>

void drawPyramid(int input);

int main(void)
{
    int input;
    do 
    {
        input = get_int("Height:\n");
    } 
    while (input > 8 || input <= 0);
    drawPyramid(input);
}

void drawPyramid(int input)
{
    for (int line = 0; line < input; line++)
    {
        for (int spaces = input - line; spaces > 1; spaces--)
        {
            printf(" ");
        }
        for (int hashes = 0; hashes < line + 1; hashes++)
        {
            printf("#");
        }
        printf("  ");
        for (int hashes = 0; hashes < line + 1; hashes++)
        {
            printf("#");
        }
        printf("\n");
    }
}
     

File: problem_sets/mario.py
import cs50

while True:
    n = cs50.get_int("Height: ")
    if n >=0 and n <= 23:
        break

for i in range(n):
    print(' ' * (n-i-1), end="")
    print('#' * (i+1), end="")
    print('  ', end="")
    print('#' * (i+1))
